lib/webvalve.rb

```diff
@@ -35,8 +35,22 @@ def config_paths
       @config_paths ||= Set.new
     end
 
+    def logger
+      if defined?(::Rails)
+        ::Rails.logger | default_logger
+      else
+        default_logger
```

Comment:
if we define this method in all cases then we will want to respect the settable `@logger` ivar in the non-Rails case. `@logger || default_logger`.

Reply:
You're right, my bad!